.azure-pipelines/azure_devops.yml

```diff
@@ -1,22 +1,47 @@
-name: "$(Year:yyyy).$(Month).$(BuildID)-ci"
-
-trigger:
-  batch: true
-  branches:
-    include:
-      - main
-
-pr: none
-
-resources:
-  repositories:
-    - repository: openjdk_azure_devops_harness
-      type: git
-      name: github_move/openjdk-azure-devops-harness
-      ref: refs/heads/main
-
-variables:
-  JTREG_VERSION: "6.1"
-
-stages:
-  - template: /ci_main.yml@devops_demo
\ No newline at end of file
+name: "$(Year:yyyy).$(Month).$(BuildID)"
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+
+pr: none
+
+resources:
+  repositories:
+    - repository: jdk-harness
+      type: git
+      name: Devdiv/jdk-harness
+      ref: refs/heads/main
+      project: "Devdiv"
+
+variables:
+  JTREG_VERSION: "6.1"
+
+stages:
+  - template: /.devops/jdk11u/release.yml@jdk-harness
+
+name: "$(Year:yyyy).$(Month).$(BuildID)-ci"

```

Comment:
What is the purpose of keeping L25 - L46?

Are there any conflicts with using multiple triggers?

Reply:
I think those are from the original file from patches
I do not have these changes in my local